crates/zed/src/languages/haskell/highlights.scm

```diff
@@ -0,0 +1,632 @@
+;; Copyright 2022 nvim-treesitter
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;     http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+; ----------------------------------------------------------------------------
+; Parameters and variables
+; NOTE: These are at the top, so that they have low priority,
+; and don't override destructured parameters
+(variable) @variable
+
+(pat_wildcard) @variable
+
+(function
+  patterns:
+    (patterns
+      (_) @variable.parameter))
```

Comment:
Unfortunately the Zed and neovim highlights files aren't compatible, as we use different conventions for syntax captures. You can find the list of captures that we support here: https://github.com/zed-industries/zed/blob/main/crates/theme/src/styles/syntax.rs

Reply:
This is a super helpful reference, thank you!